src/pages/PullRequestPage/subroute/FilesChangedTab/FilesChangedTab.spec.tsx

```diff
@@ -55,45 +85,74 @@ afterAll(() => {
 })
 
 interface SetupArgs {
-  planValue: 'team' | 'pro'
-  flagValue: boolean
+  planValue: (typeof TierNames)[keyof typeof TierNames]
+  multipleTiers: boolean
+  privateRepo: boolean
 }
 
 describe('FilesChangedTab', () => {
-  function setup({ planValue, flagValue }: SetupArgs) {
+  function setup({ planValue, multipleTiers, privateRepo }: SetupArgs) {
     mockedUseFlags.mockReturnValue({
-      multipleTiers: flagValue,
+      multipleTiers: multipleTiers,
     })
 
     server.use(
       graphql.query('OwnerTier', (req, res, ctx) => {
-        if (planValue === 'team') {
+        if (planValue === TierNames.TEAM) {
           return res(ctx.status(200), ctx.data(mockTeamTier))
         }
 
         return res(ctx.status(200), ctx.data(mockProTier))
-      })
+      }),
+      graphql.query('GetPullTeam', (req, res, ctx) =>
+        res(ctx.status(200), ctx.data(mockCompareData))
+      ),
+      graphql.query('GetRepoSettingsTeam', (req, res, ctx) =>
+        res(
+          ctx.status(200),
+          ctx.data({
+            owner: { repository: { private: privateRepo } },
+          })
+        )
+      )
     )
   }
 
-  describe('user has pro tier', () => {
-    it('renders files changed table', async () => {
-      setup({ planValue: 'pro', flagValue: false })
-      render(<FilesChangedTab />, { wrapper })
-
-      const table = await screen.findByText('FilesChanged')
-      expect(table).toBeInTheDocument()
-    })
-  })
-
-  describe('user has team tier', () => {
-    it('renders team files changed table', async () => {
-      setup({ planValue: 'team', flagValue: true })
-
-      render(<FilesChangedTab />, { wrapper })
-
-      const table = await screen.findByText('Hi')
-      expect(table).toBeInTheDocument()
-    })
-  })
+  describe.each`
+    multipleTiers | planValue          | privateRepo
+    ${true}       | ${TierNames.BASIC} | ${true}
+    ${true}       | ${TierNames.BASIC} | ${false}
+    ${true}       | ${TierNames.TEAM}  | ${false}
+    ${false}      | ${TierNames.BASIC} | ${true}
+    ${false}      | ${TierNames.BASIC} | ${false}
+    ${false}      | ${TierNames.TEAM}  | ${true}
+    ${false}      | ${TierNames.TEAM}  | ${false}
+  `(
```

Comment:
Can you make a ticket to update the other tests (team plan stuff only) so that we can hit all of these test cases